partcad-cli/src/partcad_cli/click/commands/status.py

```
@@ -52,10 +52,10 @@ def get_tar():
 
 
 def get_runtime():
-    with logging.Action("Status", "runtime"):
-        runtime_path = os.path.join(path, "runtime")
+    with logging.Action("Status", "sandbox"):
+        runtime_path = os.path.join(path, "sandbox")
         runtime_total = (get_size(runtime_path)) / 1048576.0
-        logging.info("Runtime environments size: %.2fMB" % runtime_total)
+        logging.info("Sandbox environments size: %.2fMB" % runtime_total)
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)